WTWSMS/history/titles/c_labourd.txt

```diff
@@ -38,9 +38,16 @@
 561.1.1={
 	holder = 206851 # Charibert Meroving 
 }
+565.1.1={
+	holder = 1310013 
+	liege = "k_paris"
+}
 567.1.1={
+    liege = "k_neustria"
+}
+581.1.1={
 	holder = 30021 # Chilperic Meroving 
 }
 584.1.1={
-	holder = 30029 # Chlothar II Meroving 
+	holder = 1310013
```

Comment:
He didn't keep the title, but he never got it back since he became a bishop instead.

Reply:
Ok, I'll change it